### PR TITLE
feat: #31 ref tooltips w/microtip

### PR DESCRIPTION
**Pull request by paolobrasolin**

### CODE REVIEW

File: _plugins/ref_tag.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ def render(context)
     registers = context.registers
     site = registers[:site]
     node = site.collections['nodes'].docs.detect { |doc| doc.data['slug'] == @slug }
-    "<a href='#{site.baseurl}#{node.url}' class='slug'>[#{@slug}]</a>"
+    "<a href='#{site.baseurl}#{node.url}' role='tooltip' aria-label='#{node.title}' class='slug'>[#{@slug}]</a>"
```
Comment (jonsterling) [inline _plugins/ref_tag.rb]: Should this be `node.data["title"]` or something? I am getting some deprecation warnings.

**Comment by jonsterling** [comment]
This tooltips look really nice!

**Review by jonsterling — APPROVED** [review]
Looks great, thanks!! 🚀

File: _plugins/ref_tag.rb (hunk 1 above)
Comment (paolobrasolin) [reply _plugins/ref_tag.rb]: Yes! `node.title` is good for templates where the page is serialized, but here we're accessing the `Page` instance directly. Fixed.

**Pull request merged**